src/Apps/Order/Components/PaymentMethodSummaryItem.tsx

```diff
@@ -47,7 +52,7 @@ export const PaymentMethodSummaryItemFragmentContainer = createFragmentContainer
   {
     order: graphql`
       fragment PaymentMethodSummaryItem_order on CommerceOrder {
-        paymentMethod
```

Comment:
removed this because we are using `paymentMethodDetails` to render relevant data.